### PR TITLE
partner: add media section

### PR DESCRIPTION
**Pull request by tormath1**
I didn't try it locally - I hope the image format is correct.

### CODE REVIEW

File: content/partners/media/kube_careers.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+key: kube_careers
+title: Kube Careers
+category: media
+logo: /images/partners/media/logo-color@2x.png
+website: 'https://kube.careers/'
+socials: []
+---
+
+Hand-picked Kubernetes jobs. Clear salary ranges and apply directly to companies.
```
Comment (pabluk) [inline content/partners/media/kube_careers.md:10]: I think we can use the long description version from their media kit, that could be more descriptive.

**Review by pabluk — COMMENTED** [review]
Don't forget to add `media` also on these lines:
* https://github.com/sre-summercamp/sre-summercamp.github.io/blob/0d5a7f15999b7a553a8213ef4bb2a9448b0945d2/content/_index.fr.md?plain=1#L122
* https://github.com/sre-summercamp/sre-summercamp.github.io/blob/0d5a7f15999b7a553a8213ef4bb2a9448b0945d2/content/_index.md?plain=1#L124

Regarding the images, I don't know why but the devfest-theme-hugo
 theme expects also a webp version of these images. You can convert them using `cwebp`:
 ```
 cwebp logo-color@2x.png -o logo-color@2x.webp
 ```

File: content/partners/media/kube_careers.md (hunk 1 above)
Comment (tormath1) [reply content/partners/media/kube_careers.md:10]: Nice catch, I did not notify there is `description.md` file.

**Comment by tormath1** [comment]
@pabluk thanks for the review - comments have been addressed. :)

**Review by pabluk — COMMENTED** [review]
Regarding the description, I think that could be better to just choose one of them between, because there almost similar Short description, Short description (alternative), Description and Description (alternative).

**Comment by pabluk** [comment]
> @pabluk thanks for the review - comments have been addressed. :)

no problem

**Review by pabluk — APPROVED** [review]

**Pull request merged**